src/main/resources/default.properties

```diff
@@ -0,0 +1,165 @@
+# app name
```

Comment:
I'm thinking mainly of security and authentication here. A common use case is to use, say, SAML or OpenID Connect rather than a default setup. There are a many properties here, and Spring Security is the source for the configuration right now. So yes, we could switch to environment variables, but that isn't the current source for Spring configuration. This allows us to drop in a different Spring config for one deployment, without that being the core one supplied. Essentially we'd be taking a vanilla war file from a Maven central repo, and replacing a few files for a local deployment. 

Given where we are now, with Spring Security, that's much easier than rewriting the whole of the current security configuration system to use environment variables.